libs/components/forms/src/lib/modules/file-attachment/file-item.component.ts

```diff
@@ -135,10 +137,10 @@ export class SkyFileItemComponent implements DoCheck {
   }
 
   public isFile() {
-    return this.fileItemService.isFile(this.fileItem as SkyFileItem);
+    return this.#fileItemService.isFile(this.fileItem as SkyFileItem);
```

Comment:
Possible to remove the `as SkyFileItem`?

Reply:
Done. It took a bit of extra code due to the types not being compatible - but the new code is probably safer.